docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -18,6 +18,7 @@ Person *--> Name
 Person *--> Email
 Person *--> Address
 Person *--> Course
+Enum "<<enumeration>>\nRole" as Role
```

Comment:
Could you move this line somewhere above for neatness? For example:
```suggestion
Enum "<<enumeration>>\nRole" as Role

AddressBook *-right-> "1" UniquePersonList

// other code here

Person *--> Name
Person *--> Email
Person *--> Address
Person *--> Course
Person *--> Role
Person *--> "*" Phone
```

Reply:
But if then Role box would move to the most left

Reply:
oh.. is that an issue for the diagrams? I thought the functionality would be the same

Reply:
I also thought that it would be the same but if I move "Enum "<<enumeration>>\nRole" as Role", the Role box would move to the left also. I think it is because if I move it upward, it initializes before other classes are initialized.

Reply:
ok then ill just approve